### ISSUE TITLE
[Test failure] TC_AUTHOR_50: Numerical input component generates an error with the formula shown in the usage example.

### ISSUE THREAD

**Issue by DeimerM** [openedx/wg-build-test-release#420]
### Release

Sumac

### Expected behavior

It is required that when the example formula is entered into the numerical input component, it should render without any errors.

### Actual behavior

When a numeric component is added, and the formula shown in the example is entered `m*c^2`, an error occurs

### Steps to reproduce

1. Add a numerical input component.
2. Enter the formula shown in the example: m*c^2.
3. Observe the error.

### Additional information

Error obtained:

![Screenshot From 2024-11-14 09-48-33](https://github.com/user-attachments/assets/99edcbd8-6a0e-4546-b73f-546ca5c8c2a8)

Example:

![Screenshot From 2024-11-14 09-48-51](https://github.com/user-attachments/assets/f64ffaf9-a2c6-432d-afbd-b33f0eff7224)

**Comment by mariajgrimaldi** [comment]
@DonatoBD: have you had a chance to triage this? Let me know!

**Comment by pdpinch** [comment]
I'll take a look into this, but I think the issue is that the sample problem is incomplete. If I recall correctly, the problem needs to define variables like m and c in order for the learner submission to work.

**Comment by DonatoBD** [comment]
Hi, as @pdpinch mentioned, the answer's value should include integers, fractions, and constants. The formula works correctly in nutmeg (without mfe), but it doesn't work in redwood (with mfe enabled).

I'm not very sure about the constants, but I'm reviewing ['Completing Mathematical and Scientific Assignments'](https://edx.readthedocs.io/projects/open-edx-learner-guide/en/latest/completing_assignments/SFD_mathformatting.html#completing-mathematical-and-scientific-assignments)  to see if it's a configuration issue with the component or an error with the constants we've had for a long time.

**Comment by DonatoBD** [comment]
Well, this is kinda interesting. Comparing it to nutmeg (without mfe), I could see that the olx of the "numerical input" problems adapts depending on the answer.

When we put "m*c^2" and see the olx, it's created as a "stringresponse", while if we put "10" it creates it as a "numericalresponse". 

However, in the mfe of course-authoring, the template for each problem is already defined, and the [numerical input ](https://github.com/openedx/frontend-app-authoring/blob/e75ce15a675382fdd1cb954216dcecea89f582b0/src/editors/data/constants/problem.ts#L14)only uses ["numericalresponse"](https://github.com/openedx/frontend-app-authoring/blob/e75ce15a675382fdd1cb954216dcecea89f582b0/src/editors/data/constants/basicOlxTemplates/numeric.js).

For the case of defining variables, there's the advanced problem of "math expressions", which uses ["formularesponse"](https://github.com/openedx/frontend-app-authoring/blob/e75ce15a675382fdd1cb954216dcecea89f582b0/src/editors/data/constants/advancedOlxTemplates/formularesponse.js).

This makes me think that this case has been happening since the new component editors were activated with the mfe. I don't know if we should open an issue in course-authoring so that the creation of the olx for numerical input adapts the same as before, or on the contrary, pass it to the product to see if they want to stick to the premise that it only accepts integers and for everything else, use another type of component

cc @mariajgrimaldi

**Comment by pdpinch** [comment]
Great find! 

You've reminded me why we still haven't committed to using the new problem editors.

**Comment by mariajgrimaldi** [comment]
@DonatoBD: I agree we should open the issue in the course authoring MFE and discuss what can do to move this forward there. What do you think?

**Comment by mariajgrimaldi** [comment]
@DonatoBD: friendly ping :)

**Comment by mariajgrimaldi** [comment]
Here's the issue related to this report: https://github.com/openedx/frontend-app-authoring/issues/1680

**Comment by mariajgrimaldi** [comment]
Closing in favor of https://github.com/openedx/frontend-app-authoring/issues/1680 to avoid duplication, all discussions should be had there
